core/themes/bartik/template.php

```diff
@@ -24,6 +24,33 @@ function bartik_css_alter(&$css) {
   if (theme_get_setting('color_legacy') && isset($css[$theme_path . '/css/colors.css'])) {
     $css[$theme_path . '/css/colors.css']['data'] = $theme_path . '/css/colors-legacy.css';
   }
+
+  // Remove Bartik's `/css/style.css` if using a custom breakpoint.
+  if (config_get('menu.settings', 'menu_breakpoint') == 'custom') {
+    unset($css[$theme_path . '/css/style.css']);
```

Comment:
This doesn't seem right. `style.css` in Bartik contains lots of CSS unrelated to the menu. Why would the entire CSS file be excluded if using a custom breakpoint?

Reply:
@quicksketch That file has generic CSS as well as breakpoint-specific CSS. We remove it here only when using a custom breakpoint, and then in `bartik_preprocess_page()` we include `style.menu-breakpoint.css` and `style.menu-breakpoint-queries.css` - those two files are copies of `style.css` but just splitting the breakpoint-specific stuff out separate from the generic stuff.

Reply:
We do the same thing with Basis.

Reply:
Hm, in Basis, we're copying ~100 lines of CSS that is specific to menus, it makes sense to swap out that menu CSS file with an alternative. In Bartik, this is 1700 lines of CSS (literally all the styling for the whole theme) that is getting duplicated. I think that makes a big maintenance head-ache and it's quite confusing why we'd need so much duplicate code. I'll post a comment in the main issue.